modules/c++/sys/source/OSWin32.cpp

```diff
@@ -100,7 +100,7 @@ sys::Pid_T sys::OSWin32::getProcessId() const
 
 void sys::OSWin32::removeFile(const std::string& pathname) const
 {
-    if (DeleteFile(pathname.c_str()) != true)
+    if (DeleteFile(pathname.c_str()) != TRUE)
```

Comment:
Yeah I recall a lot of this code being guilty of that.  I wouldn't bother spending the time cleaning up all of the existing functions, but for ones that you've already touched here due to the compiler warning, yeah, go ahead and change to be `if (DeleteFile(...) == FALSE)`